sql/item_mods.sql

```diff
@@ -2074,13 +2846,20 @@ INSERT INTO `item_mods` VALUES (10969,5,-10);
 INSERT INTO `item_mods` VALUES (10969,8,5);
 INSERT INTO `item_mods` VALUES (10969,9,5);
 INSERT INTO `item_mods` VALUES (10970,370,2);
+INSERT INTO `item_mods` VALUES (10971,1,12); -- STRENDU MANTLE, DEF:12 HP+110 VIT+3
+INSERT INTO `item_mods` VALUES (10971,2,110); -- HP
+INSERT INTO `item_mods` VALUES (10971,10,3); -- VIT
 INSERT INTO `item_mods` VALUES (10972,1,14);
 INSERT INTO `item_mods` VALUES (10972,10,5);
 INSERT INTO `item_mods` VALUES (10973,1,8);
 INSERT INTO `item_mods` VALUES (10973,30,5);
 INSERT INTO `item_mods` VALUES (10973,233,1);
 INSERT INTO `item_mods` VALUES (10974,1,9);
 INSERT INTO `item_mods` VALUES (10974,31,5);
+INSERT INTO `item_mods` VALUES (10975,1,17); -- ARCHON CAPE, DEF:17 Occasionally annuls severe physical damage taken Dark weather:Accuracy+13 Attack+13
+-- INSERT INTO `item_mods` VALUES (10975,OCCASIONALLY ANNULS SEVERE PHYSICAL DMG TAKEN,1);
+-- INSERT INTO `item_mods` VALUES (10975,DARK WEATHER:ACC,13);
+-- INSERT INTO `item_mods` VALUES (10975,DARK WEATHER:ATK,13);
```

Comment:
These two in item_latents.sql - use the same latent condition and latent param as diabolos's pole (latent weather, param dark weather)